Mage/src/main/java/mage/abilities/common/PlayCardTriggeredAbility.java

```diff
@@ -0,0 +1,105 @@
+package mage.abilities.common;
+
+import mage.abilities.TriggeredAbility;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.effects.Effect;
+import mage.constants.TargetController;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+
+// Author: alexander-novo
+// A triggered ability for cards which say "whenever <someone> play(s) a card..."
+public class PlayCardTriggeredAbility extends TriggeredAbilityImpl {
+
```

Comment:
Can also be used for [[Juju Bubble]]

Reply:
Cool thanks. I had problems finding cards that needed to use this.